app/views/admin/questions/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= f.label(:title) %>
   <%= f.text_field(:title, required: true, class: "rounded-md xl:w-3/4") %>
 
-  <%= f.label(:subtitle) %>
+  <%= f.label(:subtitle, class:'mt-3') %>
```

Comment:
Remove the subtitle field from the form and relevant pages. Backend do not support it

Reply:
The subtitle fields you have mentioned are on Questions pages not on Blogs Page. It is already added by you on questions pages but not properly styled. I just styled them.